packages/playwright/src/index.ts

```diff
@@ -258,7 +258,7 @@ const playwrightFixtures: Fixtures<TestFixtures, WorkerFixtures> = ({
       onApiCallBegin: (data: ApiCallData) => {
         const testInfo = currentTestInfo();
         // Some special calls do not get into steps.
-        if (!testInfo || data.apiName.includes('setTestIdAttribute') || data.apiName === 'tracing.groupEnd')
+        if (!testInfo || data.apiName.includes('setTestIdAttribute') || data.apiName === 'tracing.groupEnd' || data.apiName.startsWith('apiResponse.'))
```

Comment:
The proper way is to wrap all methods of `APIResponse` class with `_wrapApiCall(func, true)`. That will account for tracing, steps and debug logging.

Reply:
That only exists for channels though, and `APIResponse` isn't an interface inside protocol.yml, but only an object

Reply:
smart!